go/service/policies/api_op_ListPolicies_test.go

```diff
@@ -77,10 +77,10 @@ func Test_ListPoliciesInternalServerError(t *testing.T) {
 	apiClient, err := New(*cfg)
 	assert.NoError(t, err, "failed to initialize api client")
 
-	policies, apiError, err := apiClient.ListPolicies(ListPoliciesInput{})
+	policies, err := apiClient.ListPolicies(ListPoliciesInput{})
 
-	assert.NoError(t, err, "failed to list policies.")
 	assert.Nil(t, policies)
-	assert.NotNil(t, apiError)
-	assert.Equal(t, len(apiError), 1)
+	assert.NotNil(t, err)
+
+	assert.Contains(t, err.Error(), internal.ACTION_API_SUCCESSFUL_RESULT, "error message should contain 'connector api error'")
```

Comment:
I didn't get this part

Reply:
it just checks that the error is from the connector and not an internal one like "failed to build http request" or "failed to marshal json"